snippets/measurements_of_interest_summary.sql

```diff
@@ -50,17 +50,21 @@ SELECT
   COUNTIF(value_as_concept_id IS NOT NULL
       AND value_as_concept_id != 0) AS num_concept_values,
   COUNTIF(operator_concept_id IS NOT NULL) AS num_operators,
+  IF(src_id = "PPI/PM", "PPI", "EHR") AS measurement_source,
   measurement_concept_id,
   unit_concept_id
 FROM
   `{CDR}.measurement`
 INNER JOIN
-  labs_of_interest
-USING
-  (measurement_concept_id, unit_concept_id)
+ labs_of_interest USING(measurement_concept_id, unit_concept_id)
+LEFT JOIN
+  `{CDR}.measurement_ext` USING(measurement_id)
+WHERE
+  person_id IN ({COHORT_QUERY})
```

Comment:
Curious if you know off-hand: how does this stack up against a JOIN? Naively I imagine this is slower for large ID-sets, but certainly BigQuery could be optimizing it.

Reply:
'JOIN' is a short way of saying [INNER JOIN](https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#inner-join).

In this case, the performance should be identical because we expect a 1-to-1 mapping between rows in `measurements` and `measurements_ext`. But we use a LEFT JOIN since it would be acceptable to have nulls on the right if the `measurement_ext` table were missing any rows (missing any mappings for measurement ids).

Reply:
Thanks - I should have been more explicit, I was asking about

`WHERE IN` (used here), vs `JOIN`

Reply:
Ah, got it. I don't know off-hand, but I would go with your assumption that it is optimized.